### PR TITLE
Updated request detail page

### PR DESCRIPTION
**Pull request by laulauland**
Resolves FP-3823

Revisiting the request detail view with more expansive and expanded defaults, minimap, and other things

https://github.com/fiberplane/fpx/assets/17225208/568f681f-625b-43ed-9142-fbd06bef2671

### CODE REVIEW

File: frontend/src/pages/RequestDetailsPage/TextJsonViewer.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    nordTheme["--w-rjv-background-color"] = "transparent";
+    return (
+      <JsonView
+        value={json}
+        style={nordTheme}
+        displayDataTypes={false}
+        collapsed={collapsed}
+      />
+    );
+  } catch (error) {
```
Comment (brettimus) [inline frontend/src/pages/RequestDetailsPage/TextJsonViewer.tsx:20]: note to self: test this with a text response

**Comment by brettimus** [comment]
A few notes:

- I modified the layout/grid and made things a little more responsive
- Spacing is a little funky, but we can fix that later
- Should test with _a lot_ of events to make sure overflow doesn't get weird in the minimap
- Minimap gets confused quite easily, probably quite a few edge cases to work through

**Review by brettimus — APPROVED** [review]
